src/components/SRNEntry.vue

```diff
@@ -179,50 +180,51 @@ export default {
         },
       ];
     },
-    /* checks to see if the input character is a number. Makes use of ASCII values.
-    @param {Event} - e - event triggered when a character is typed.  */
+    /** Checks to see if the input character is a number. Makes use of ASCII values.
+     * @param {Object} e - event triggered when a character is typed
+     */
     isValidNumericEntry(e) {
       if (e.keyCode >= 48 && e.keyCode <= 57) return true;
       else e.preventDefault();
     },
-    // adds a new dict to the array - userIDList
+    /** Adds a new dict to the array - userIDList */
     addNewEmptyField() {
       this.userIDList.push({ userID: "", valid: false });
     },
-    /* removes an element in the array at a given index. 
-    @param {Number} - index - the index of the input box where - button is clicked. */
+    /** Removes an element in the array at a given index.
+     * @param {Number} index - the index of the input box where - button is clicked
+     */
     removeInputField(index) {
       this.userIDList.splice(index, 1);
     },
-    // sets the invalid input message to null (default)
+    /** Sets the invalid input message to null (default) */
```

Comment:
```suggestion
    /** Resets the invalid input message */
```
It's not setting it to `null` to be precise, it's setting it to an empty string. Also, by not mentioning what it's setting the value to be, we save ourselves the trouble of changing the docstring again in the future if our default value changes from an empty string to, let's say, `null`